components/app-configs/mlde/workspace_config/mlde-task-template.yaml

```diff
@@ -0,0 +1,16 @@
+environment:
+  environment_variables: {}
+  image: nolanc/pyspark-nb:spark3.5-rapids1.12-azure3.4-autosparksession0.0.2
```

Comment:
This value will likely change in the future. I can replace with a placeholder, or leave it as it'll work for now.